Automattic/jetpack
# Jetpack Recommendations: Add dashboard banner

#### Changes proposed in this Pull Request:

This PR adds a banner for the Jetpack Recommendations

<img width="1258" alt="image" src="https://user-images.githubusercontent.com/42627630/106941131-0955dd00-66e8-11eb-888c-87062f66c7c3.png">

<img width="399" alt="image" src="https://user-images.githubusercontent.com/42627630/106947624-3b6b3d00-66f0-11eb-9f4e-9302c88bf87f.png">

#### Jetpack product discussion



MT: p1HpG7-asW-p2
Design: p6TEKc-4cJ-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






1. On a new site visit wp-admin. Verify that the dashboard banner displays.
2. Select some options and click continue. If you chose store, verify that you are taken to the WooCommerce prompt. If not, verify that you are taken to the Monitor prompt.
3. Return to the wp-admin dashboard and verify that the dashboard banner does not display.
4. Reset the Jetpack options and go the wp-admin dashboard. Dismiss the banner and reload the page, verifying that it does not display.
5. Reset the Jetpack options and go to `/wp-admin/admin.php?page=jetpack#/recommendations`. Advance past the site-type question and then visit the wp-admin dashboard. Verify that the banner does not display.

#### Proposed changelog entry for your changes:


Recommendations: added a dashboard banner to help users discover the best Jetpack features.
